### PR TITLE
Sensorhead twist msg

### PR DESCRIPTION
**Pull request by Joschi3**
The sensorhead teleop plugin is now able to send either twist or orientation msgs. The behavior by changing the value of the parameter use_twist_msg.

### CODE REVIEW

**Comment by Martin-Oehler** [comment]
Please resolve the conflicts first.

**Comment by Joschi3** [comment]
All conflicts are now resolved.

File: hector_joy_teleop_plugins/config/sensorhead/sensorhead_common.yaml [hunk 1]
```diff
@@ -11,4 +11,6 @@ sensorhead_min_pan: -1.57
 
 sensorhead_tilt_inverted: false
 
-sensorhead_command_topic: "camera/command"
\ No newline at end of file
+sensorhead_command_topic: "camera/command"
+
+use_twist_command : false
```
Comment (Martin-Oehler) [inline hector_joy_teleop_plugins/config/sensorhead/sensorhead_common.yaml]: You can enable using the twist command here. It should not have any drawbacks.

File: hector_joy_teleop_plugins/src/sensorhead_teleop.cpp [hunk 2]
```diff
@@ -81,28 +110,35 @@ void SensorheadTeleop::forwardMsg(const sensor_msgs::JoyConstPtr& msg)
...
+            sensorhead_pan_ = 0;
+            sensorhead_tilt_ = 0;
+            publishCommand();// use quaternion msg to reset orientation
         }
 
     }
 }
 
 
+void SensorheadTeleop::publishCommand()
```
Comment (Martin-Oehler) [inline hector_joy_teleop_plugins/src/sensorhead_teleop.cpp]: Maybe rename to `publishOrientationCommand` to differentiate it more from `publishTwistCommand`.

**Review by Martin-Oehler — CHANGES_REQUESTED** [review]

**Pull request merged**